Return metadata for each item returned by a metadata query (#778)
box/box-java-sdk: CHANGELOG.md

# Changelog

## Next Release

> - Return metadata for each item returned by a metadata query

Reviewer: ```suggestion
- Add metadata to each item returned by a metadata query
```
